images/macos/macos-10.15-Readme.md

```diff
@@ -114,14 +123,15 @@ date: Week 6
 - NUnit 3.6.1
 
 ### Xcode
-| Version                        | Build                          | Path                           |
-| ------------------------------ | ------------------------------ | ------------------------------ |
-| 11.3.1 (default)               | 11C505                         | /Applications/Xcode_11.3.1.app |
-| 11.3                           | 11C29                          | /Applications/Xcode_11.3.app   |
-| 11.2.1                         | 11B500                         | /Applications/Xcode_11.2.1.app |
-| 11.2                           | 11B52                          | /Applications/Xcode_11.2.app   |
-| 11.1                           | 11A1027                        | /Applications/Xcode_11.1.app   |
-| 11.0                           | 11A420a                        | /Applications/Xcode_11.app     |
+| Version                           | Build                             | Path                              |
+| --------------------------------- | --------------------------------- | --------------------------------- |
+| 11.4                              | 11N111s                           | /Applications/Xcode_11.4_beta.app |
```

Comment:
could you please manually add `(beta)` word here so it will be clear that we installed Xcode 11.4 Beta?
https://github.com/microsoft/azure-pipelines-image-generation/pull/1481/files#diff-e9f9822df5b3b9087dd94d4c3c083277R126 as an example.